pkg/driverbuilder/builder/templates/archlinux.sh

```diff
@@ -37,6 +34,6 @@ modinfo {{ .ModuleFullPath }}
 {{ if .BuildProbe }}
 # Build the eBPF probe
 cd {{ .DriverBuildDir }}/bpf
-make LLC=/usr/bin/llc-7 CLANG=/usr/bin/clang-7 CC=/usr/bin/gcc KERNELDIR=/tmp/kernel
+make LLC=/usr/bin/llc-7 CLANG=/usr/bin/clang-7 CC=/usr/bin/gcc-{{ .GCCVersion }} KERNELDIR=/tmp/kernel
```

Comment:
I think we can also cleanup `CC=` from the various bpf build scripts, as it is useless!

Reply:
Applied everywhere :heavy_check_mark:

Reply:
That's a super quick response time :rocket: